server/src/main/java/org/elasticsearch/search/aggregations/bucket/prefix/InternalIpPrefix.java

```diff
@@ -318,15 +318,30 @@ public Bucket createBucket(InternalAggregations aggregations, Bucket prototype)
         );
     }
 
+    public Bucket createBucket(Bucket prototype, InternalAggregations aggregations, long docCount) {
```

Comment:
If this is only used in this method I think it should be private. I think other aggs have something like this be public but I don't know why. I think it was a historical accident.

Reply:
Of course.